testnets/injectivetestnet/assetlist.json

```diff
@@ -30,6 +30,91 @@
         }
       ],
       "type_asset": "sdk.coin"
+    },
+    {
+      "description": "USDC is a USD-pegged stablecoin available on the Injective testnet.",
+      "denom_units": [
+        {
+          "denom": "factory/inj17vytdwqczqz72j65saukplrktd4gyfme5agf6c/usdc",
+          "exponent": 0
+        },
+        {
+          "denom": "USDC",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/inj17vytdwqczqz72j65saukplrktd4gyfme5agf6c/usdc",
+      "name": "USD Coin",
+      "display": "USDC",
+      "symbol": "USDC",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.svg"
+      },
+      "coingecko_id": "usd-coin",
+      "images": [
+        {
+         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.svg"
+        }
+      ],
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "OSMO is the native token of the Osmosis blockchain, bridged over to the Injective testnet.",
+      "denom_units": [
+        {
+          "denom": "ibc/2CD6478D5AFA173C86448E008B760934166AED04C3968874EA6E44D2ECEA236D",
+          "exponent": 0
+        },
+        {
+          "denom": "UOSMO",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/2CD6478D5AFA173C86448E008B760934166AED04C3968874EA6E44D2ECEA236D",
+      "name": "Osmosis",
+      "display": "UOSMO",
+      "symbol": "UOSMO",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/osmo.png"
+      },
+      "coingecko_id": "osmosis",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/osmo.png"
+        }
+      ],
+      "type_asset": "ics20"
```

Comment:
for ics20 assets, please define the `traces` array (in this case pointing to osmosistestnet)